src/page-components/homepage/index.tsx

```diff
@@ -10,6 +10,10 @@ interface IndexPageProps {
 }
 
 const IndexPage: React.FC<IndexPageProps> = ({ data }: IndexPageProps) => {
+  React.useEffect(() => {
```

Comment:
2 věci: 

- `useEffect` bych importoval napřímo
- možná by stálo za to extrahovat do `useScrollToTopOnRender`